<launch>
<arg name="gui" default="True" />
<param name="robot_description" textfile='$(find urdf_robot)/urdf/dynamic_robot.urdf' />
<param name="use_gui" value="$(arg gui)"/>
<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
<node name="FkSolverKDL" pkg="urdf_robot" type="FkSolverKDL" />
<node name="FkSolverManual" pkg="urdf_robot" type="FkSolverManual" />
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find urdf_robot)/rviz/my_robot.rviz" required="true" />
</launch>

